firebase-vertexai/gradle.properties

```diff
@@ -12,5 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-version=16.0.0-beta06
+version=16.0.0-beta10
```

Comment:
necessary?

Reply:
Oops, left that in from testing maven local